imagersite/imager_images/templates/album_add.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
+{% load i18n %}
+
+{% block content %}
+<nav id='album_add'>
+{% if user.is_authenticated %}
```

Comment:
One hopes that it's not possible to get to this template without being authenticated.  I'll check in `views.py` below